ctapipe/reco/HillasReconstructor.py

```diff
@@ -99,13 +99,67 @@ class that reconstructs the direction of an atmospheric shower
 
     """
 
-    def __init__(self, config=None, parent=None, **kwargs):
+    def __init__(self, subarray, config=None, parent=None, **kwargs):
         super().__init__(config=config, parent=parent, **kwargs)
-        self.hillas_planes = {}
-        self.divergent_mode = False
-        self.corrected_angle_dict = {}
+        self.subarray = subarray
+
+        self.cam_R_m = {
+            tel_id: tel.camera.geometry.guess_radius()
```

Comment:
this doesn't follow naming rules, and probably shouldn't be a public property, so I suggest renaming to `self._cam_radius_m` (and same for `self._cam_radius_deg`)